config/config.go

```diff
@@ -48,8 +48,12 @@ const (
 	osPatchEnabledDefault     = false
 	debugEnabledDefault       = false
 
-	osPatchStateFileWindows = `C:\Program Files\Google\OSConfig\osconfig_patch.state`
-	osPatchStateFileLinux   = "/etc/osconfig/osconfig_patch.state"
+	configDirWindows        = `C:\Program Files\Google\OSConfig`
+	configDirLinux          = "/etc/osconfig"
```

Comment:
if this is going to have 'config', ok, but if it's only going to have a marker file, we should be creating that in /, /var/tmp or similar.

Reply:
Agreed, I think storing it in temp would be better

Reply:
Just to make it clearer, I just moved the restart file to /tmp and c:\windows\temp, the config dir is still used for state like it was previously was , but that should probably moved to /var/local